pkg/phlare/modules.go

```diff
@@ -329,21 +329,15 @@ func (f *Phlare) initRing() (_ services.Service, err error) {
 
 func (f *Phlare) initStorage() (_ services.Service, err error) {
 	objectStoreTypeStats.Set(f.Cfg.Storage.Bucket.Backend)
-	if cfg := f.Cfg.Storage.Bucket; cfg.Backend != "filesystem" {
```

Comment:
I think we should keep this for now as this won't work because of https://github.com/grafana/pyroscope/issues/2029

Reply:
I'll check if I can resolve it.